.travis.yml

```diff
@@ -1,7 +1,18 @@
 language: python
-python:
-    - '3.6'
-    - '3.7'
+matrix:
+  include:
+    - python: "3.6"
+    - python: "3.7"
+    - python: "3.8"
+    - python: "3.8-dev"
+    - python: "3.9-dev"
+    - python: "pypy3"
+      env: NO_MYPY=true
+  allow_failures:
+    - python: "3.8-dev"
+    - python: "3.9-dev"
+    - python: "pypy3"
+      env: NO_MYPY=true
```

Comment:
```suggestion
```
The environment variable should be removed here as well, since the block in `allow_failures` should match what's in `include`.